scripts/versions.sh

```diff
@@ -4,7 +4,7 @@
 
 # Set up the versions to be used
 AWM_RELAYER_VERSION=${AWM_RELAYER_VERSION:-'v0.2.2'}
-SUBNET_EVM_VERSION=${SUBNET_EVM_VERSION:-'v0.5.6'}
+SUBNET_EVM_VERSION=${SUBNET_EVM_VERSION:-'v0.5.8'}
 # Don't export them as they're used in the context of other calls
-AVALANCHE_VERSION=${AVALANCHE_VERSION:-'v1.10.10'}
+AVALANCHE_VERSION=${AVALANCHE_VERSION:-'v1.10.14'}
```

Comment:
Does it work to go straight to AvalancheGo v1.10.15 to avoid and extra update?

Reply:
We can update it to v1.10.15. But in the go.mod, we still need to use v1.10.14 since subnet-evm v0.5.8 uses v1.10.14. Do we want keep the same version?

Reply:
Right. Lets keep `v1.10.14` for now 👍 

We'll later update both subnet-evm and avlanchego to the latest versions, which hopefully don't have any further breaking changes.